src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -29,13 +29,24 @@ def _environment_variables_type(value):
     return {'name': env_name, 'value': env_value}
 
 
+def _secure_environment_variables_type(value):
+    """Space-separated values in 'key=value' format."""
+    try:
+        env_name, env_secure_value = value.split('=', 1)
+    except ValueError:
+        message = ("Incorrectly formatted secure environment settings. "
+                   "Argument values should be in the format a=b c=d")
+        raise CLIError(message)
+    return {'name': env_name, 'secureValue': env_secure_value}
+
+
 secrets_type = CLIArgumentType(
     validator=validate_secrets,
     help="space-separated secrets in 'key=value' format.",
     nargs='+'
 )
 
-
+# pylint: disable=too-many-statements, too-many-branches
```

Comment:
I can understand `too-many-statements` where does the `too-many-branches` come from?

Reply:
copy and paste error. FIxed.